src/current/v23.1/explain-analyze.md

```diff
@@ -33,7 +33,16 @@ Parameter          | Description
 
 ## Required privileges
 
-The user requires the appropriate [privileges]({% link {{ page.version.version }}/security-reference/authorization.md %}#managing-privileges) for the statement being explained.
+To generate a statement bundle, you must have the [privileges]({% link {{ page.version.version }}/security-reference/authorization.md %}#managing-privileges) required by the statement. The user requires the appropriate [privileges]({% link {{ page.version.version }}/security-reference/authorization.md %}#managing-privileges) for the SQL statement being explained.
```

Comment:
I think the following might be slightly clearer:

"
To generate a statement bundle, you must have the privileges required to execute the SQL statement, as well as the privileges required to collect the statement bundle.
"

There are really 2 sets of privileges required when capturing the statement bundle using `EXPLAIN (ANALYZE)`.

Reply:
Done, thank you!